gadgets/testing/testing.go

```diff
@@ -16,15 +16,33 @@ package testing
 
 import (
 	"os"
+	"os/exec"
 	"testing"
 
 	"github.com/cilium/ebpf/rlimit"
 	"github.com/moby/moby/pkg/parsers/kernel"
 	"github.com/stretchr/testify/require"
 
 	utilstest "github.com/inspektor-gadget/inspektor-gadget/internal/test"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/testing/utils"
 )
 
+const (
+	K8sDistroAKSAzureLinux  = "aks-AzureLinux"
+	K8sDistroAKSUbuntu      = "aks-Ubuntu"
+	K8sDistroARO            = "aro"
+	K8sDistroMinikubeGH     = "minikube-github"
+	K8sDistroEKSAmazonLinux = "eks-AmazonLinux"
+	K8sDistroGKECOS         = "gke-COS_containerd"
+)
+
+func SkipK8sDistro(t testing.TB, distro string) {
```

Comment:
Might come in handy in that test with multiple distro skips
```suggestion
func SkipK8sDistro(t testing.TB, distros ...string) {
```